### PR TITLE
Nilber Remon  - Spirit Island: Chronicles of Anahuac

### PR DESCRIPTION
**Pull request by neuhelos**

### CODE REVIEW

**Comment by helisian** [comment]
Inspired by the board game 'Spirit Island' [copyrighted]

File: textBasedAdventure.js [hunk 1]
```diff
@@ -1,5 +1,146 @@
...
-
 let nameInput = readline.question("Enter your name: ")
+console.log(`${nameInput}! Behold the beguiling lands of Spirit Island.`)
+
+readline.keyInPause("Ready to commit yourself and listen to the history of our present.")
+
+readline.keyInPause("In the endless cresting waves on our shores and enshrined in the rhizomatic matrices of (in)organic pulses threading the soil beneath, we are those who have no face and abide by the dynamic equilibrium of our ancestral lands, and its Free People. We are the Guardians. And we face a new pestilence arriving upon its hallowed ground")
+
+function pause(milliseconds) {
+    var start = new Date().getTime();
```
Comment (CelineChole) [inline textBasedAdventure.js]: Use `let` instead of `var`

File: textBasedAdventure.js [hunk 2]
```diff
@@ -1,5 +1,146 @@
...
+
+pause (2000)
+
+///////////////////////////////////////////////////////////////////////////////////////////////
+//GAME SET-UP
+
+// Initial Health Indices
+let spiritHealth = 150
+let settlerCapitalHealth = 400
+//let settlerCapitalStructuralHealth = 400
```
Comment (CelineChole) [inline textBasedAdventure.js]: Make sure you remove commented code before submitting your project

File: textBasedAdventure.js [hunk 3]
```diff
@@ -1,5 +1,146 @@
...
+        let year = 1
+        for (let i = 0; i <= p; i++) {
+            let popInc = p0 + (p0 * (percent/100)) + aug
+            if (popInc < p) {
+                year++
+                p0 = popInc
+            }  
+        } 
+        return year
+    }
```
Comment (CelineChole) [inline textBasedAdventure.js]: I think I know this one! ;) Make sure you remove any code that is not linked to the project.

File: textBasedAdventure.js [hunk 4]
```diff
@@ -1,5 +1,146 @@
...
+    } else if (colonialOutpost[targetOutpost] === colonialOutpost[1]){
+        guardianPower()
+        settlerDamage()
+    }
+}
+
+
+const spiritDamage = () => { 
+    if (attack === settlerCapitalPower){
+    spiritHealth += settlerCapitalPower
```
Comment (CelineChole) [inline textBasedAdventure.js]: This line should be indented

File: textBasedAdventure.js [hunk 5]
```diff
@@ -1,5 +1,146 @@
...
+    for (var i = 0; i < 1e7; i++) {
+      if ((new Date().getTime() - start) > milliseconds){
+        break;
+      }
+    }
+  }
+
+
+let spiritName
+const becomeGuardian = () => {let joinFight = readline.keyInYNStrict("Will you join us in our struggle against the menace of colonization? ")
```
Comment (CelineChole) [inline textBasedAdventure.js]: Well done for using keyInYNStrict 👍

**Pull request closed without merging**